dotnet/src/Connectors/Connectors.Memory.CosmosMongoDB/Connectors.Memory.CosmosMongoDB.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- THIS PROPERTY GROUP MUST COME FIRST -->
-    <AssemblyName>Microsoft.SemanticKernel.Connectors.AzureCosmosDBMongoDB</AssemblyName>
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.CosmosMongDB</AssemblyName>
```

Oops, I already approved the PR, but this one is important :)
```suggestion
    <AssemblyName>Microsoft.SemanticKernel.Connectors.CosmosMongoDB</AssemblyName>
```

---

Good catch! Will definitely fix.